web/src/javascript/drawMap.js

```diff
@@ -8,8 +8,8 @@ var yScreenPosOnMap = 200;
 
 const mapCanvas = document.createElement('canvas');
 const mapContext = mapCanvas.getContext('2d');
-const mapWidth = screenWidth*3;
-const mapHeight = screenHeight*3;
+export const mapWidth = screenWidth*3;
+export const mapHeight = screenHeight*3;
```

Comment:
I unsure of the exact meaning of your sentence, but : If you don't export something it's impossible to reach it outside of this file.